Sources/OktaIdx/Internal/Implementations/Version1/IDXClientAPIv1.swift

```diff
@@ -153,6 +153,60 @@ extension IDXClient.APIVersion1: IDXClientAPIImpl {
             completion(IDXClient.Response(api: self, v1: response), nil)
         }
     }
+    
+    func redirectResult(with context: IDXClient.Context, redirect url: URL) -> IDXClient.RedirectResult {
+        guard let redirect = Redirect(url: url),
+              let originalRedirect = Redirect(url: configuration.redirectUri) else {
+            return .invalidRedirectUrl
+        }
+        
+        guard originalRedirect.scheme == redirect.scheme && originalRedirect.path == redirect.path else {
```

Comment:
Can you put these two expressions on separate lines, for consistency?

Reply:
Done